plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -694,6 +696,63 @@ public void testTimestampColumnAndTimestampWithTimeZoneConstant()
         }
     }
 
+    @Test
+    public void testAccessDeniedToTableWithExtraCredentialImpersonation()
+            throws Exception
+    {
+        String user = "test_access_denied_to_table_" + randomTableSuffix();
```

Comment:
nit: add `user` to the actual name. it's easier to debug in db when you know that this entity is a user.